components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -648,14 +694,38 @@ class NewTabPage extends React.Component<Props, State> {
         imageSrc={this.imageSource}
         imageHasLoaded={this.state.backgroundHasLoaded}
         colorForBackground={colorForBackground}
+        hasSponsoredRichMediaBackground={hasSponsoredRichMediaBackground}
         data-show-news-prompt={((this.state.backgroundHasLoaded || colorForBackground) && this.state.isPromptingBraveNews && !defaultState.featureFlagBraveNewsFeedV2Enabled) ? true : undefined}>
         <OverrideReadabilityColor override={ this.shouldOverrideReadabilityColor(this.props.newTabData) } />
         <BraveNewsContextProvider>
         <EngineContextProvider>
+
+        {
+          this.sponsoredRichMediaBackgroundInfo &&
+          <SponsoredRichMediaBackground
+              sponsoredRichMediaBackgroundInfo={this.sponsoredRichMediaBackgroundInfo}
+              richMediaHasLoaded={this.state.backgroundHasLoaded}
+              onLoaded={() => {
+                if (!this.state.backgroundHasLoaded) {
+                  this.setState({ backgroundHasLoaded: true })
+                }
+              }}
+              onEventReported={(adEventType: NewTab.SponsoredRichMediaEventType) => {
+                triggerSponsoredRichMediaAdEvent(this.props.newTabData.brandedWallpaper, adEventType)
+
+                if (this.sponsoredRichMediaBackgroundInfo && adEventType === 'click') {
+                  window.open(this.sponsoredRichMediaBackgroundInfo.targetUrl, '_self', 'noopener,noreferrer');
```

Comment:
cc @thypon do we allow `window.open` on `chrome://` urls? For some reason I thought we didn't?